### DISCUSSION TITLE
Self-hosting: Use mounted folder as storage?

### DISCUSSION THREAD

**Discussion by geodose** [jgraph/drawio#3259] [Q&A] [answered]
Is it possible to use a mounted network share as a storage target? I don't see how to do this via the docker documentation.

**Comment by davidjgraph** [comment, marked as answer]
No, there's no back-end for this. The app is built for cloud storage. Webdav is probably the closest thing.

**Reply by geodose** [reply]
Thank you for the response, good to know!
